umh-core/examples/example-config-redpanda.yaml

```diff
@@ -0,0 +1,23 @@
+# Copyright 2025 UMH Systems GmbH
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+agent:
+  metricsPort: 8080
+
+redpanda:
+  name: redpanda
+  desiredState: active
+  redpandaServiceConfig:
+    retentionMs: 1000000
+    retentionBytes: 1000000000
```

Comment:
for retention to properly work we must also set the segment size right? because retention only works on closed segments

Reply:
Default segment size is 1073741824 byte (~1073 MB).
Imo this should be fine..